docs/gen_ref_pages.py

```diff
@@ -4,7 +4,9 @@
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
+
 # http://www.apache.org/licenses/LICENSE-2.0
+
```

Comment:
```suggestion
#
# http://www.apache.org/licenses/LICENSE-2.0
#
```
Not 100% sure, but there is a chance this might trigger an error in the license header check.